docs/src/design/story/wireframes/v1_3/styles.css

```
@@ -0,0 +1,741 @@
+  font-family: var(--mono); font-size: 10px; color: var(--ink-3);
+}
+.device-chrome .dots { display: flex; gap: 4px; }
+.device-chrome .dot { width: 10px; height: 10px; border-radius: 50%; border: 1px solid var(--ink); }
+.device-chrome .url {
+  flex: 1;
+  border: 1px solid var(--ink);
+  padding: 1px 8px;
+  background: var(--paper);
+  text-align: left;
+}
+.device-body { }
```

Stylelint (17.11.1)

[error] Empty block (block-no-empty)

(block-no-empty)